packages/office-ui-fabric-react/src/components/Tooltip/docs/TooltipBestPractices.md

```diff
@@ -0,0 +1,18 @@
+### Content
+
+- Don’t use a tooltip to restate a button name that’s already shown in the UI.
+- When a control or UI element is unlabeled, use a simple, descriptive noun phrase. For example: “Highlighting pen.” Only capitalize the first word (unless a subsequent word is a proper noun), and don’t use a period.
+- For a disabled control that could use an explanation, provide a brief description of the state in which the control will be enabled. For example: “This feature is available for line charts.”

```

Comment:
```suggestion
- For a disabled control that could use an explanation, provide a brief description of the state in which the control will be enabled. For example: "This feature is available for line charts".

```

Reply:
In this case, there is guidance for using a period if the tooltip is a complete sentence. So the punctuation is part of the example and the quotation mark is correct.